examples/email_analytics_reporting/Ballerina.toml

```diff
@@ -0,0 +1,14 @@
+[package]
+org = "thakshakar"
+name = "bulk_update_reply_to_email"
+version = "0.1.0"
+distribution = "2201.11.0"
+
+[build-options]
+observabilityIncluded = true
+
+[[dependency]]
+org = "ballerinax"
+name = "hubspot.crm.engagements.email"
+version = "1.0.0"
+repository = "local"
```

Comment:
We can remove these here, because after this connector is released, it will import the module from the central directly